.env.example

```diff
@@ -1,23 +1,46 @@
-APP_ORIGIN
+APP_ORIGIN=localhost:3000
+
+# This is what's used to create a DOI upon publication, without it a DOI is not created
+# This is only needed if you are testing the publication of modules
 CROSSREF_LOGIN_ID
 CROSSREF_LOGIN_PASSWD
 CROSSREF_URL
-DATABASE_URL
+
+# Postgres DB URL. Default values found in ./docker-compose.yml
+# This is needed to do any local development
+DATABASE_URL="postgres://dbuser:dbpassword@localhost:5432/researchequals-dev"
+
+
+# Used for connecting Ocrid account to user. Can setup credentials for free by creating an account: https://orcid.org/ and looking at developer settings
+# This is only needed if you are testing/linking users to their Orcid accounts
 ORCID_CLIENT_ID
 ORCID_CLIENT_SANDBOX
 ORCID_CLIENT_SECRET
+
+# Used as the email client
+# This is needed to send emails
 POSTMARK_TOKEN
-UPLOADCARE_LOCALE
+
+# Freemium software for file storage, https://uploadcare.com/
 UPLOADCARE_PUBLIC_KEY
 UPLOADCARE_SECRET_KEY
+
+# Freemium software, used for free text searching, https://https://www.algolia.com/
 ALGOLIA_APP_ID
 ALGOLIA_API_SEARCH_KEY
 ALGOLIA_API_ADMIN_KEY
 ALGOLIA_PREFIX
+
+# Used for cronjobs, https://quirrel.dev/
+# This is not needed for local development
```

Comment:
```suggestion
# Used for cronjobs and queues
# https://github.com/quirrel-dev/quirrel
# This is not needed for local development
```

The service itself is discontinued so you need to self-deploy this for production 😿